src/lib/netlist/devices/nld_4042.cpp

```diff
@@ -0,0 +1,100 @@
+// license:GPL-2.0+
+// copyright-holders:Couriersud, Jonathan Gevaryahu
+/*
+ * nld_4042.cpp
+ *
+ *  CD4042BM/CD4042BC Quad Clocked D Latch
+ *
```

Comment:
See last comment on documentation (CD4029)

Reply:
Similar to cd4029, merged into nlm_cd4xxx_lib.cpp